src/internal/m365/graph/errors.go

```diff
@@ -227,7 +227,7 @@ func hasErrorCode(err error, codes ...errorCode) bool {
 		return false
 	}
 
-	var oDataError *odataerrors.ODataError
```

Comment:
but who wouldn't want to (sometimes) use an `***<concrete type>`? 😛

Reply:
wtb `*nil`